js/ui/components/clubhouse.js

```diff
@@ -114,12 +115,33 @@ var ClubhouseNotificationSource = new Lang.Class({
     },
 });
 
+var NotifiableButton = new Lang.Class({
+    Name: 'NotifiableButton',
+    Extends: St.Button,
+
+    _init: function(params) {
+        this.parent(params);
+
+        this.connect('clicked', () => {
+            this._stopPulsing();
+        })
+    },
+
+    pulsate: function() {
+        this.get_children()[0].add_style_pseudo_class('highlighted');
+    },
+
+    _stopPulsing: function() {
```

Comment:
Make the function public so we call from outside, like the `pulsate` one. In fact, I'd simply create a function `setHighlighted(true|false)`, which simplifies the code.